tools/report-converter/tests/unit/sparse_output_test_files/sample.out

```diff
@@ -0,0 +1,5 @@
+files/sample.c:4:1: warning: symbol 'machine_id' was not declared. Should it be static?
+./files/sample.h:3:40: warning: incorrect type in argument 2 (different address spaces)
+files/sample.c: note: in included file:
+./files/sample.h:3:40: warning: incorrect type in argument 1 (different address spaces)
+./files/sample.h:3:40:    expected struct task_struct *p1
```

Comment:
This seems to be invalid sample output. Sparse usually outputs expected and got. Are you sure that sparse outputs what you added in this file?

Reply:
Did you actually run sparse on this sample file? I just cannot imagine that sparse outputs this on the sample file header after reading the sample file header.

Reply:
Yes Lukas I did add these files to the base of the Linux kernel. But sadly it did not produce any warning/errors when I ran sparse over it. 

Moreover, we just use the `.c` files in the tests to provide the `.c` file as an input to the parser instead of the `.out` file and check if it is invalid. We don't compile it at run time. Hence I decided to produce the sample output file manually and make the `sample.expected.plist` file accordingly. If you want I can add a GOT line in the `sample.out` and modify the expected plist accordingly

Reply:
Hmm, but that really seems like a broken test here. I expect that we have the source code as test .c and .h that really produces the warnings with sparse for the output you want to test here.